src/overlays/actors/ovl_En_S_Goro/z_en_s_goro.c

```diff
@@ -4,18 +4,72 @@
  * Description: Goron in Goron Shrine / Bomb Shop Goron
  */
 
+/*
+This actor appears to be used in three scenes.
+-- Goron Shrine: Winter
+       Three variants (one instance of each used in game).
+       Main behavior determined by whether the Goron Elder's Son is asleep, and whether they themselves have been put to
+sleep. One of them also has a dialogue tree that checks whether you have lit the chandelier (at least if the kid is not
+crying)
+-- Goron Shrine: Spring
+       Six variants - one instance of each appearing in a circle around the elder on the upper level of the shrine if
+you've beaten Goht during that cycle. Dialogue determined by whether you have talked to elder in that context as a goron
+at least once during that cycle. They have a cheering animation used while the elder is talking to you, otherwise they
+stand idle.
+-- Bomb Shop
+       Only one variant. Goron who sells you powder kegs in the bomb shop.
+*/
+
+/*
+Week Event Flags:
+        22 Bit 2	--	Goron Elder's Son has been pacified (Winter)
+        77 Bit 7	--	You have spoken to the Goron Elder in goron form during spring
+
+        36 All Bits
+        37 Bits 0 - 2
+                Track whether or not you have ever spoken to each of the three winter shrine gorons
+                in various forms. ("Ever" being since last time reset, of course)
+*/
+
 #include "z_en_s_goro.h"
+#include "overlays/actors/ovl_En_Gk/z_en_gk.h" //Goron Elder's Son
+#include "overlays/actors/ovl_En_Jg/z_en_jg.h" //Goron Elder
+#include "objects/object_taisou/object_taisou.h"
 
 #define FLAGS (ACTOR_FLAG_1 | ACTOR_FLAG_8 | ACTOR_FLAG_10)
-
 #define THIS ((EnSGoro*)thisx)
 
+#define EYETEX_INDEX_MAX 3
+#define EN_S_GORO_ROLLEDUP_YOFFSET 14.0f
```

Comment:
```suggestion
#define EN_S_GORO_EYETEX_INDEX_MAX 3
#define EN_S_GORO_ROLLEDUP_YOFFSET 14.0f
```
I would keep the actor prefix